etrago/cluster/electrical.py

```diff
@@ -403,6 +454,23 @@ def delete_ehv_buses_no_lines(network):
 
 
 def ehv_clustering(self):
+    """
+    Cluster the network based on Extra High Voltage (EHV) grid.
+
+    If `network_clustering_ehv` argument is True, the function clusters the
+    network based on the EHV grid. If `network_clustering` argument is not 
+    active, it calls the `load_shedding` function to apply load shedding.
```

Comment:
This load_shedding stuff changed again, which is already on dev. Should we adjust the documentation?